test/parallel/test-http2-server-destroy-before-shutdown.js

```diff
@@ -0,0 +1,44 @@
+// Flags: --expose-http2
+'use strict';
+
+const common = require('../common');
+if (!common.hasCrypto) common.skip('missing crypto');
```

Comment:
Nit: can you please put the body on the next line? I think it is the dominant style throughout the code base.

Reply:
Done!